tools/e2e-commons/bin/e2e-env.sh

```diff
@@ -53,6 +54,20 @@ gb_setup() {
 	$BASE_CMD wp plugin activate gutenberg
 }
 
+# Exports E2E WP password from config file.
+export_e2e_config() {
+	if [[ ! -f 'config/local.cjs' ]]; then
+		echo "Decrypted config file not found! Have you run 'pnpm config:decrypt' yet?";
+		exit 1
+	elif [[ -z "$WP_FORCE_ADMIN_USER" ||  -z "$WP_FORCE_ADMIN_PASS" ]]; then
```

Comment:
I might still skip the part allowing the caller environment to override it. Is there any case we wouldn't want the values from the config?
```suggestion
	else
```

Reply:
The idea was less to allow caller environments to override it, and more to ensure we didn't spin up node just to parse that file multiple times. Both are fairly theoretical, though, so I can live with your change. :^)